internal/pinning.go

```diff
@@ -1,23 +1,21 @@
-package ebpf
+package internal
 
 import (
 	"errors"
 	"fmt"
 	"os"
-
-	"github.com/cilium/ebpf/internal"
 )
 
-func pin(currentPath, newPath string, fd *internal.FD) error {
+func Pin(currentPath, newPath string, fd *FD) error {
```

Comment:
We don't want to export this API.

Reply:
Since this in `internal` it's not reachable outside of the package, take a look at https://golang.org/doc/go1.4#internalpackages

Reply:
Ah! TIL - Thx!